addons/trenches/stringtable.xml

```diff
@@ -222,5 +226,220 @@
             <Chinesesimp>移除掩体中</Chinesesimp>
             <Chinese>移除掩體中</Chinese>
         </Key>
+        <Key ID="STR_ACE_Trenches_PlaceCamouflage">
+            <English>Place camouflage</English>
+            <German>Tarnung platzieren</German>
+            <Russian>Камуфлировать окоп</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_PlaceCamouflageProgress">
+            <English>Placing camouflage...</English>
+            <German>Platziere Tarnung...</German>
+            <Russian>Камуфлирую окоп...</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_RemoveCamouflage">
+            <English>Remove camouflage</English>
+            <German>Tarnung entfernen</German>
+            <Russian>Убрать камуфляж</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_RemoveCamouflageProgress">
+            <English>Removing camouflage...</English>
+            <German>Entferne Tarnung...</German>
+            <Russian>Убираю камуфляж окопа...</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_EnvelopeGiantName">
+            <English>Trench (Envelope - Giant)</English>
+            <German>Schützengraben (Rund - Sehr groß)</German>
+            <Russian>Огромный окоп</Russian>
```

Comment:
"Large"

Reply:
Whats the problem with giant?

Reply:
Sounds like cartoony. Wouldn't use before Large and Huge.

Reply:
okay I will change it to Large